ci/bazel-scripts/diff.sh

```diff
@@ -14,7 +14,8 @@ set -x
 cd "$(git rev-parse --show-toplevel)"
 
 MERGE_BASE="${MERGE_BASE_SHA:-HEAD}"
-COMMIT_RANGE=${COMMIT_RANGE:-$MERGE_BASE".."}
+# we can't use HEAD here because that is the merge commit which contains the changes of the current HEAD of master
+COMMIT_RANGE=$MERGE_BASE".."${BRANCH_HEAD_SHA:-}
```

Comment:
This would be safer:
```suggestion
COMMIT_RANGE="$MERGE_BASE..${BRANCH_HEAD_SHA:-}"
```